### DISCUSSION TITLE
bac0.writeValue do not seem to be working

### DISCUSSION THREAD

**Discussion by levky18t** [ChristianTremblay/BAC0#483] [Q&A]
Tried to use the format of doing a writeValue as described in the documentation: https://bac0.readthedocs.io/en/stable/read.html#write-to-property

after running the writeValue function, I am not getting any errors. But when doing a readValue at the same object/property, the value has not changed. It is still the same value as before running the writeValue function.

Anyone else running into a similar issue?

**Comment by ChristianTremblay** [comment]
Doc are still needing a lot of rework after the migration to async.

bacnet.write call the creation of a asyncio task
if you need control over the write and be sure it's completed, you can use `await bacnet._write(args)`
